app/views/deploys/_deploy.html.erb

```diff
@@ -11,7 +11,7 @@
       <% end %>
     </td>
     <td>
-      <%= link_to deploy.summary, [@project || deploy.project, deploy] %>
+      <%= link_to deploy.summary, [@project || deploy.project, deploy, with_deleted: deploy.deleted? ? true : nil] %>
```

Comment:
I though it would be `soft_deleted_at?`

Reply:
It is only deleted?, as per soft_delete gem.